meta-facebook/at-cb/src/platform/plat_hook.c

```diff
@@ -288,12 +288,135 @@ ina233_init_arg ina233_init_args[] = {
 	},
 };
 
+sq52205_init_arg accl_pwr_monitor_sq52205_init_args[] = {
```

Comment:
nit: There seems to be a mixture of tabs and spaces in this variable.  Can we get them all switched to spaces and consistent indentation?

Reply:
Hi @williamspatrick .
Modify with your suggestion, please help to review this commit again, thank you.

Regards,
Lora.